libs/domain/checkout/src/deprecated-reexports.ts

```diff
@@ -1,11 +1,11 @@
-import * as services from '@spryker-oryx/checkout/services';
 import { featureVersion } from '@spryker-oryx/utilities';
+import * as services from '../services/src/index';
 
 const reexports: typeof services =
   featureVersion < '1.1' ? services : (undefined as any);
 
 /** @deprecated since 1.1, use DefaultCartService from @spryker-oryx/cart/services */
```

Comment:
`use DefaultCartService from @spryker-oryx/cart/services` we should use Cart service instead of Checkout? Is it just a copy-paste?

Reply:
Can you also check the whole file, there are a lot of mentions of `@spryker-oryx/cart/services`?

Reply:
👍  thanks, fixed